pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -349,13 +351,21 @@ func TestReconciler(t *testing.T) {
 		Request(corev1.ResourceCPU, "1").
 		Image("", nil)
 
+	baseWPCWrapper := utiltesting.MakeWorkloadPriorityClass("test-wpc").
+		WorkloadPriorityValue(100)
+
+	basePCWrapper := utiltesting.MakePriorityClass("test-pc").
+		PriorityValue(200)
+
 	cases := map[string]struct {
 		focus             bool
 		reconcilerOptions []jobframework.Option
 		job               batchv1.Job
 		wantJob           batchv1.Job
 		workloads         []kueue.Workload
 		wantWorkloads     []kueue.Workload
+		wpc               kueue.WorkloadPriorityClass
+		pc                schedulingv1.PriorityClass
```

Comment:
maybe they can be in a single place `extraObjs []client.Object`

Also, try to put all the inputs separate from the outputs (`wantWorkloads`, `wantErr`)